doc/related_projects.rst

```diff
@@ -302,3 +302,22 @@ Domain specific packages
 
 - `MSMBuilder <http://msmbuilder.org/>`_  Machine learning for protein
   conformational dynamics time series.
+
+Translations of scikit-learn documentation
+------------------------------------------
+
+The only official documentation for scikit-learn is the current English
+version, however some translations are available aiming to improve scikit-learn
+dissemination among non-native English speaking. Those efforts are community
```

Comment:
I think "speaking" is not enough here. Either `speaking communities` or `speakers`?